main.tf

```diff
@@ -0,0 +1,245 @@
+/**
```

Comment:
Why do we need a separate `main.tf` file here? How does this reconcile with the `main.tf` in the `terraform-google-forseti` repo?  What if there are changes in `terraform-google-forseti` repo, then do we have to manually keep this up to date?

Reply:
main.tf in the top directory has the resources that needs to be created to deploy Forseti. main.tf in integration_tests/fixtures/forseti has the resources that needs to be created to run tests. 

The code in `forseti-security/main.tf` needs to be updated only if there are changes in the way Forseti is deployed

Reply:
`main.tf` in the root directory has been removed.